### PR TITLE
Files list: Add "Show in Explorer" popup when nothing under mouse cursor

### PR DESCRIPTION
**Pull request by Krakean**
Extension to my previous PR #13105, but now "Show in Explorer" added too. Small addition, hope it make it to 3.0 before feature freeze :)

Screenshot: https://i.imgur.com/kZEptSm.png

### CODE REVIEW

File: editor/filesystem_dock.cpp [hunk 1]
```diff
@@ -1448,12 +1448,13 @@ void FileSystemDock::_files_list_rmb_select(int p_item, const Vector2 &p_pos) {
...
 void FileSystemDock::_rmb_pressed(const Vector2 &p_pos) {
-	folder_options->clear();
-	folder_options->set_size(Size2(1, 1));
+	file_options->clear();
+	file_options->set_size(Size2(1, 1));
 
-	folder_options->add_item(TTR("New Folder.."), FOLDER_NEW_FOLDER);
-	folder_options->set_position(files->get_global_position() + p_pos);
-	folder_options->popup();
+	file_options->add_item(TTR("New Folder.."), FOLDER_NEW_FOLDER);
```
Comment (akien-mga) [inline editor/filesystem_dock.cpp]: Now that we're using `file_options`, should it be `FILE_NEW_FOLDER`? :) I'm not sure what's the difference between the two.

Comment (Krakean) [reply editor/filesystem_dock.cpp]: Hm, true. Will fix it now.

**Comment by MillionOstrich** [comment]
You should be able to call directly into FileSystemDock::_files_list_rmb_select() (or, better still, move the contents of that method into yours and have that method call you).

- remove the `if (files->get_item_text(p_item) == "..") {` block as this should not execute anymore because the .. is unselectable now
- you will have to move the add seperator calls to the end of the blocks they are in so that they aren't added erroneously.
- rename the methods following the refactor to be a bit more clear

**Comment by Krakean** [comment]
> You should be able to call directly into FileSystemDock::_files_list_rmb_select() (or, better still, move the contents of that method into yours and have that method call you).
This method is not called by default when you RMB on file list when nothing under cursor, need to modify list_item to send corresponding signal. 
But decided to implement separate signal, to make my patch better for reviwing and merge and less test.
You can change my PR they way you think it will be better, after merge, but I suppose for clarity the way how it is made currently is fine enough.

**Comment by Krakean** [comment]
@akien-mga Seems like rebased correctly this time, check please :)

**Comment by MillionOstrich** [comment]
I liked the idea of the separate signal 👍 - I was thinking you could get one signal's handler to just be an adaptor to call to the others handler which would save a duplicating a bit of code. It isn't a huge deal, as the way it is right now is pretty clear anyway, just something to consider!

**Comment by Krakean** [comment]
>It isn't a huge deal, as the way it is right now is pretty clear anyway, just something to consider!

Understand your suggestions, thought about it, but was in hurry due to Feature Freeze, so wanted made my patch as much clear to integrate as possible and produce 0 possible issues :)

**Pull request merged**